static/app/views/issueDetails/groupDetails.tsx

```diff
@@ -284,13 +317,9 @@ function useFetchGroupDetails({
   const organization = useOrganization();
   const router = useRouter();
   const params = router.params;
-  const location = useLocation();
   const {projects} = useProjects();
-  const allGroups = useLegacyStore(GroupStore);
-  const group = allGroups.find(({id}) => id === params.groupId) as Group;
 
   const [project, setProject] = useState<Project | null>(null);
```

Comment:
Something that you can do now (here or in a followup PR) is remove this state variable and replace it with something like this:

```
const project =
    projects?.find(({id}) => id === groupData?.project?.id) ?? groupData?.project ?? null;
```

Less state is always good and this means the project will be filled in without an extra render cycle.